### PR TITLE
*: fix minor typos

### PR DESCRIPTION
**Pull request by gyuho**
/cc @xiang90 @heyitsanthony

### CODE REVIEW

File: Documentation/v2/api_v3.md [hunk 1]
```diff
@@ -1,92 +0,0 @@
-# etcd3 API
-
-TODO: API doc
-
-## Data Model
```
Comment (xiang90) [inline Documentation/v2/api_v3.md]: why this is removed?

Comment (gyuho) [inline Documentation/v2/api_v3.md]: Do we keep this for v3 preview? Then I will revert this change.

Comment (xiang90) [inline Documentation/v2/api_v3.md]: yes.

**Comment by gyuho** [comment]
@xiang90 Just reverted. Only minor typos. PTAL.

Thanks.

**Comment by xiang90** [comment]
lgtm

**Pull request merged**